sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -172,6 +213,31 @@ def test_big_query_read_new_types(self):
           use_standard_sql=True)))
       assert_that(result, equal_to(expected_data))
 
+  def test_big_query_read_none_values(self):
```

Comment:
Please add `@attr('IT')`. I think nose requires it to run filter test scenarios to include into our integration test suites. I am surprised to see that this test actually ran without this decorator... Perhaps because  there is an IT marker on a test with the same prefix `test_bigquery_read`... Anyway, it would be better to either move this decorator to be a class decorator or to add it to every test method.